homework-projects/eszter_prekopova/using_photocollage.adoc

```diff
@@ -0,0 +1,25 @@
+[[using_photocollage]]
+= Using PhotoCollage
+
+By uploading the input photos, you generate a big poster. *PhotoCollage* automatically arranges the photos to fill the whole poster. Afterwards, you can change the final layout, dimensions, border, or swap photos in the generated grid.
+
+image::layout.png[Layout]
+
+The algorithm generates random layouts that place photos while taking advantage of all free space. It tries to fill all space while keeping each photo as large as possible. You can generate new layouts until finding one that suits you. The color and width of the borders are changeable.
+
+*PhotoCollage* is currently available in 10 languages:
+
+* [.mark]#English#
```

Comment:
This is a valid syntax however the end result looks the same as if you'd write `* English`

I'm curious because I rarely use these tags and I'm not familiar with [.mark] Is this what you wanted to achieve?